Duffel.ApiClient/DuffelApiClient.cs

```diff
@@ -42,10 +42,10 @@ public DuffelApiClient(HttpClient httpClient, string accessToken)
             _httpClient.BaseAddress = new Uri("https://api.duffel.com");
 
             _httpClient.DefaultRequestHeaders.Add("User-Agent",
-                $"Duffel/beta {executingAssemblyName.Name}/{executingAssemblyName.Version}");
+                $"Duffel/v1 {executingAssemblyName.Name}/{executingAssemblyName.Version}");
```

Comment:
the client might still be in beta?

Reply:
The client is in `beta` version, according to the releases [1]. Similar for other clients.

[1] https://github.com/duffelhq/duffel-api-dotnet/releases

Reply:
Also, I spoke with Norberto, and his plan is for that part to represent the API version from the request headers.